data/json/items/armor.json

```diff
@@ -6977,6 +6977,23 @@
     },
     {
         "type" : "ARMOR",
+        "id" : "locket_lucy",
```

Comment:
has no actual use especially considering we already have a number of items of jewelry